packages/react-dom/package.json

```diff
@@ -27,12 +27,15 @@
     "README.md",
     "index.js",
     "server.js",
-    "node-stream.js",
     "test-utils.js",
     "unstable-native-dependencies.js",
     "cjs/",
     "umd/"
   ],
+  "browser": {
```

Comment:
It might be nicer to do this with a more public browser entry point instead of the internal package details (personally I'd prefer not exposing those as separate files at all and just inline them).

With a public entry point it's at least possible to manually require the browser version if you need to. This encourages require to the internal implementation details instead.